secret-squirrel.js

```diff
@@ -8,14 +8,14 @@ module.exports = {
 	strings: {
 		deny: [],
 		denyOverrides: [
-			'62a990fd8dce4a27aafb006b58783f66@sentry.io', // browser/bundles/o-errors.js:11
-			'62a990fd8dce4a27aafb006b58783f66', // browser/bundles/o-errors.js:11|11
+			'/circleci/project/github/RedSparr0w/node', // README.md:1
+			'62a990fd8dce4a27aafb006b58783f66@sentry\\.io', // browser/bundles/o-errors/index.js:13
+			'62a990fd8dce4a27aafb006b58783f66', // browser/bundles/o-errors/index.js:13|13
```

Comment:
how comes these have changed?

Reply:
My guess is that it never worked ..?

Reply:
This previous PR somehow wasn't checked by Secret Squirrel. Possibly a github UI change?

https://github.com/Financial-Times/n-ui/commit/fcd4667bb1d736ca69d3682abc21242bd47c5e05#diff-04c6e90faac2675aa89e2176d2eec7d8